ILRepack/Steps/ResourceProcessing/BamlStreamCollector.cs

```diff
@@ -54,6 +79,22 @@ public void Process(EmbeddedResource embeddedResource, ResourceWriter resourceWr
             }
         }
 
+        private void PatchGenericThemesBaml(ResourceWriter resourceWriter)
+        {
+            byte[] existingGenericBaml;
```

Comment:
you don't make use of this byte[], is it for future use or should it be removed ?